package.json

```diff
@@ -32,7 +32,7 @@
     "@fastify/pre-commit": "^2.0.2",
     "@types/node": "^20.1.0",
     "connect-mongo": "^5.0.0",
-    "connect-redis": "^7.0.0",
+    "connect-redis": "7.0.0",
```

Comment:
I locked it because benchmark was not running due to following error
```
TypeError: redisStoreFactory is not a function
```
But, when I re-checked it now, I realized that I should have lock this version to `^6` rather than `7.0.0`  because the error still happening in `7.0.0` version.

Reply:
The benchmark is fixed in #248 :)